conf/layer.conf

```diff
@@ -3,12 +3,19 @@ BBPATH .= ":${LAYERDIR}"
 
 # We have recipes-* directories, add to BBFILES
 BBFILES += "${LAYERDIR}/recipes-*/*/*.bb \
-            ${LAYERDIR}/recipes-*/*/*.bbappend \
             ${LAYERDIR}/images/*.bb"
 
+BBFILES += " \
+./recipes-multimedia/gstreamer/gstreamer1.0-plugins-ugly_%.bbappend \
+./recipes-connectivity/openssh/openssh_8%.bbappend\
```

Comment:
Does these .bbappend show up for you when you run `bitbake-layers show-appends | grep ugly` or grep by `ssh`?

Reply:
Now it does, see new commit.